pkg/assets/walkers.go

```diff
@@ -12,35 +12,66 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package walkers
+package assets
 
 import (
 	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
 	"path/filepath"
-
-	"github.com/pkg/errors"
-
-	"github.com/kinvolk/lokomotive/pkg/assets"
 )
 
-func CopyingWalker(path string, newDirPerms os.FileMode) assets.WalkFunc {
+// CopyingWalker returns a WalkFunc which writes the given file to disk.
+func CopyingWalker(path string, newDirPerms os.FileMode) WalkFunc {
```

Comment:
I'm not sure if that's the right time to touch it, but it seems to me, that term "Walker" is an implementation detail, which shouldn't be part of the assets public API.

I'd like to have something like `assets.Read` or `assets.Unpack`, this would make other code more readable.

Reply:
I'm fine with looking into this (but in a new issue please).

Reply:
Created https://github.com/kinvolk/lokomotive/issues/796 :+1: